build_projects/update-dependencies/update-dependencies.csproj

```diff
@@ -13,6 +13,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.DotNet.VersionTools" Version="$(VersionToolsVersion)" />
+    <PackageReference Include="Nuget.Versioning" Version="4.5.0" />
+    <PackageReference Include="Nuget.Packaging.Core" Version="4.5.0" />
```

Comment:
Consider making a dependency property for this and combining with the ref added in https://github.com/dotnet/cli/commit/a4040c9983928d4aef2f82842e16dbb6aa7b8948?